src/components/common/overflow-menu/index.tsx

```diff
@@ -0,0 +1,113 @@
+import Text from '../text';
+import Link from 'next/link';
+import { MdManageAccounts, MdLogout, MdAccountCircle } from 'react-icons/md';
+import { VscGithubAlt } from 'react-icons/vsc';
+import { BiWalletAlt } from 'react-icons/bi';
+import { useState, useRef } from 'react';
+import { cn } from 'utils';
+import Card from '../card';
+const OverflowMenu = props => {
+    const buttonRef = useRef();
```

Comment:
Why is this ref needed?

Reply:
@hreyesm 
to close the menu after clicking on the button when the menu is open